src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -348,34 +343,46 @@ public void OnVolumeChanged(float volume)
 		await seekToSemaphoreSlim.WaitAsync(token);
 
 		seekToTaskCompletionSource = new();
-
 		try
 		{
```

Comment:
This is where I need help with Async/Await and flow control. `Player.RemoveListener(this)` and then adding it back fixes issue. But it needs to be added back in two different places depending on whether user scrubs timeline or developers does in code. If you scrub timeline in player itself the code needs to be in another place after `seekToSemophorSlim.Release()` for code to behave correctly. But that breaks the current flow control and causes unintended behavior. So any thoughts would be appreciated. If anyone can suggest a better way it would be great.